src/overlays/actors/ovl_En_Rr/z_en_rr.c

```diff
@@ -1,15 +1,77 @@
+/*
+ * File: z_en_rr.c
+ * Overlay: ovl_En_Rr
+ * Description: Like Like
+ */
+
 #include "z_en_rr.h"
+#include "vt.h"
 
 #define FLAGS 0x00000435
 
 #define THIS ((EnRr*)thisx)
 
+#define RR_MESSAGE_SHIELD (1 << 0)
+#define RR_MESSAGE_TUNIC (1 << 1)
+#define RR_MOUTH 4
+#define RR_BASE 0
+
+typedef enum EnRrReachState {
```

Comment:
Use value comments (as hex) instead of duplicating the name to the `enum`, imo.
That will ensure proper vertical formatting, and it's helpful anyway so I think it should be the default for all enums. Also relevant to the other enums.
```suggestion
typedef enum {
```